packages/sdk/odahuflow/sdk/clients/api_aggregated.py

```diff
@@ -107,6 +109,10 @@ def build_client(resource: OdahuflowCloudResourceUpdatePair, api_client: RemoteA
         return ModelPackagingClient.construct_from_other(api_client)
     elif isinstance(resource.resource, PackagingIntegration):
         return PackagingIntegrationClient.construct_from_other(api_client)
+    elif isinstance(resource.resource, FeedbackModelFeedbackRequest):
+        return FeedbackClient.construct_from_other(api_client)
+    elif isinstance(resource.resource, UserInfo):
+        return UserInfoClient.construct_from_other(api_client)
```

Comment:
I think it does not make sense to add new entities to this function.
Main purpose of this function is to build clients that will be used in `apply` function to batch/bulk editing.
But FeedbackClient or UserInfoClient do not implement inferface that `apply` function expects. 

So I suggest remove this 4 lines